### DISCUSSION TITLE
What * hidden * Features does Wavelog have?

### DISCUSSION THREAD

**Discussion by NoobKeksTV** [wavelog/wavelog#800] [General]
hi,
working with WaveLog for a while and stumbling across some lines in the changelog & certain issues made me wonder...
What other "hidden" features does wavelog have, that are not "openly" shown.

For example, i didnt know there was a OQRS embed feature untill i stumbled across the issue talking about it.
Over my few weeks running wavelog i also "found" features that werent that obvious too and only were found after some more "looking arround" but then after finding were pretty obvious... Like the Webpage-Map dedicated to all QSO's, the OQRS Page thingy, etc.

Maybe WaveLog hides some more Widgets like the OQRS one, that we "dont obviously" know about yet? :p

(im sorry if this is documented somewhere, i searched and searched and havent found any. idk if im just blind or they truly are hidden... or there just couldnt be any more, lol)

**Comment by HB9HIL** [comment]
Hello @NoobKeksTV 

I converted this issue to a discussion, as in fact this is neither a bug report, nor a feature request. 

We do not have a list of "hidden" features. Some of the features in Wavelog are just not properly documented in our Wiki (we know that we have a lack in documentation). 

Wavelog ist just extremely rich in features and useful eastereggs, that you may have to use Wavelog a bit longer to know everything. 😄 

You could try press `Alt + L` (Windows) or `Control + L` (Mac) to see one of those eastereggs 👍

**Comment by NoobKeksTV** [comment]
Oh 😮

Sorry, didnt even know github has this discussion feature 😳😅

Eh yeah, i noticed that a bit... 
What i just figured out (idk if its in the wiki), you provide a sample .htaccess file which already does the index.php rewrite stuff. Was trying to get it working and found that sample file already includes everything i need instead of me having to manually do create it.

What was left for me to do is to change 
`$config['index_page'] = 'index.php';`
to
`$config['index_page'] = '';`


What im missing a bit, since we have a OQRS Widget alredy, something like that ClubLog thingy showing the past x QSO's to implement on pages like qrz . Maybe even with a little OQRS request bar in the that same widget? (well.. pretty much like clublog's widget does)

or if were at it, a whole "description" page to the stations on the public pages. Like a qrz page but on my wavelog public page to share, etc. 😅


Great Software you have here. Amazing. 
In the past i only worked in Contest's and used WinContest to log everything. Since it doesnt really suite the day-to-day logging, and someone in my DOK got us a WL instance for our DOK Call, i looked into this and its just plain amazing. Everything i looked for, Web based, easy to use.
Amazing. 


(Maybe a Import from the .edi files from WinContest & some other software would be nice? currently i use a edi to adif converter to import. its not hard but a bit... idk)

**Reply by HB9HIL** [reply]
You bring various ideas and stuff. You could create for any ideas a new discussion in the "Ideas" thread.

**Reply by NoobKeksTV** [reply]
> You bring various ideas and stuff. You could create for any ideas a new discussion in the "Ideas" thread.

You mean for every one of those? :o

**Reply by HB9HIL** [reply]
> > You bring various ideas and stuff. You could create for any ideas a new discussion in the "Ideas" thread.
> 
> You mean for every one of those? :o

Maybe for the for you most important one? :D

**Reply by NoobKeksTV** [reply]
> > > You bring various ideas and stuff. You could create for any ideas a new discussion in the "Ideas" thread.
> > 
> > 
> > You mean for every one of those? :o
> 
> Maybe for the for you most important one? :D

ooo... i have so much more thingys and "little gimmicks" that would be usefull at time that come in my head there 😅

does WV automaticly check ClubLog for Confirmed QSO? It shows that it uploaded them but for example on those which have been comfirmed by the other station, it doesnt show that on the page, even after a sync?

And:
What about "automaticly" getting a default Locator from say qrz.com or other services?  If i enter a location, say Barcelona, it can load the Locator from that. But i mean like Callsign based automatic loading?

**Reply by R1BLH** [reply]
> does WV automaticly check ClubLog for Confirmed QSO? It shows that it uploaded them but for example on those which have been comfirmed by the other station, it doesnt show that on the page, even after a sync?

Yes.
![изображение](https://github.com/user-attachments/assets/8032ce26-22a1-47d6-bd5e-f0c45556caad)
![изображение](https://github.com/user-attachments/assets/e6084d9b-2bcd-401b-ac20-8ce3fe39637b)

**Reply by R1BLH** [reply]
> What about "automaticly" getting a default Locator from say qrz.com or other services? If i enter a location, say Barcelona, it can load the Locator from that. But i mean like Callsign based automatic loading?

This has already been done. For qrz.com you need a paid subscription, or you can set up a lookup from HamQTH.com.

**Reply by NoobKeksTV** [reply]
> > What about "automaticly" getting a default Locator from say qrz.com or other services? If i enter a location, say Barcelona, it can load the Locator from that. But i mean like Callsign based automatic loading?
> 
> This has already been done. For qrz.com you need a paid subscription, or you can set up a lookup from HamQTH.com.

oooohhhh. i looked into the cron and saw it does :O just saw my first  confirmed qso from clublog today. idk why the others do not update :/
maybe because i manually uploaded the "old" log to clublog before i found that auto upload in wl?

**Comment by AndreasK79** [comment]
Another semi-hidden function is ALT-Q :)

**Reply by R1BLH** [reply]
> Another semi-hidden function is ALT-Q :)

Thank you!
What other Easter eggs are there? 
Alt+Q is very helpful. I used to open the QSO info from the logbook to calculate QRB, then the QRB calculation form.

**Reply by HB9HIL** [reply]
The are called "easter eggs" because you have to find them xD haha

Serious, Wavelog is extremely rich in features and stuff. Try and find out :) Go through the Wiki. You may find some hints. 

But before you damage your keyboard. There are only Alt + L and Alt + Q at the moment ;-)

**Reply by R1BLH** [reply]
> But before you damage your keyboard. There are only Alt + L and Alt + Q at the moment ;-)

:) I would like to quote a famous quote from a classic Soviet comedy, which is on everyone's lips here in Russia, but it can't be appreciated without knowing the context of the movie.
